GameEngine/GameEngine/Engine/Systems/RenderCameraSystem.cpp

```diff
@@ -75,7 +158,12 @@ namespace AGE
 			Frustum spotlightFrustum;
 			spotlightFrustum.setMatrix(glm::perspective(spot->getCutOff() / 2.0f, spot->getExponent(), 0.1f, 1000.0f)* glm::inverse(spotEntity->getLink().getGlobalTransform()));
 
-			_scene->getBfcBlockManagerFactory()->cullOnChannel(BFCCullableType::CullableMesh, spotDrawableList->meshs, spotlightFrustum);
+			LFList<BFCItem> list;
+			_scene->getBfcBlockManagerFactory()->cullOnChannel(BFCCullableType::CullableMesh, list/*spotDrawableList->meshs*/, spotlightFrustum);
+			while (list.getSize() > 0)
+			{
+				spotDrawableList->meshs.push_back(list.pop()->getDrawable()->getDatas());
+			}
 			spotLightList.push_back(spotDrawableList);
```

Comment:
The culling of spotlights is not good I think because the frustum is not good.
That's why : 
- I asked @paulbaron to fix the computation of the frustum #502
- I asked @paulbaron to extract lines from frustum and camera for debug purpose  #522